spec/controllers/topics_controller_spec.rb

```diff
@@ -738,6 +738,22 @@ def topics_controller_show_gen_perm_tests(expected, ctx)
       expect(IncomingLink.count).to eq(1)
     end
 
+    it "doesn't renders the print view by default" do
+      user = Fabricate(:user)
+      get :show, topic_id: topic.id, slug: topic.slug, print: true
+
+      expect(response).to be_forbidden
+    end
+
+    it 'renders the print view when enabled' do
+      SiteSetting.max_prints_per_hour_per_user = 10
+
+      user = Fabricate(:user)
```

Comment:
You can probably extract this to `let(:user) { Fabricate(:user) }` so that it can be shared between both tests.